doc/endpoint/static.md

```diff
@@ -1,11 +1,20 @@
 # Serving static content
 
 Tapir contains predefined endpoints, server logic and server endpoints which allow serving static content, originating
-from local files or application resources. These endpoints respect etags as well as if-modified-since headers.
+from local files or application resources. These endpoints respect etags, byte ranges as well as if-modified-since headers.
+
+> ⚠️ **Warning** Since Tapir 1.3.0, static content is supported via the new `tapir-files` module. If you're looking for
```

Comment:
[minor] for callouts we typically use RST boxes, e.g. https://github.com/softwaremill/tapir/blob/53e3a588267d15fa012c1ee8ec99a90ca082be54/doc/docs/openapi.md#L247